tests/conftest.py

```diff
@@ -270,9 +287,11 @@ def folder_data_object():
 
 
 @pytest.fixture
-def aiida_local_code_bash(aiida_local_code_factory):
-    """Return a `Code` configured for the bash executable."""
-    return aiida_local_code_factory(executable="bash", entry_point="bash")
+def aiida_local_code_bash(aiida_code_installed):
```

Comment:
The new fixtures have `aiida_code_installed` instead of `aiida_local_code_factory`.